1_fetch/in/Legend_NLCD_land_Cover.csv

```diff
@@ -0,0 +1,21 @@
+NLCD_value,NLCD_description,Reclassify_match,Reclassify_description
+11,"Open Water - areas of open water, generally with less than 25% cover of vegetation or soil.",1,Water
+12,"Perennial Ice/Snow - areas characterized by a perennial cover of ice and/or snow, generally greater than 25% of total cover.",NA,NA
+21,"Developed, Open Space - areas with a mixture of some constructed materials, but mostly vegetation in the form of lawn grasses. Impervious surfaces account for less than 20% of total cover. These areas most commonly include large-lot single-family housing units, parks, golf courses, and vegetation planted in developed settings for recreation, erosion control, or aesthetic purposes.",4,Developed Areas
+22,"Developed, Low Intensity - areas with a mixture of constructed materials and vegetation. Impervious surfaces account for 20% to 49% percent of total cover. These areas most commonly include single-family housing units.",4,Developed Areas
+23,"Developed, Medium Intensity - areas with a mixture of constructed materials and vegetation. Impervious surfaces account for 50% to 79% of the total cover. These areas most commonly include single-family housing units.",4,Developed Areas
+24,"Developed High Intensity - highly developed areas where people reside or work in high numbers. Examples include apartment complexes, row houses and commercial/industrial. Impervious surfaces account for 80% to 100% of the total cover.",4,Developed Areas
```

Comment:
following the comment above and our previous discussion, 21 and 22 can be merged, and so can 23 and 24

Reply:
Ok clear I will fix that.